sdk/identity/identity/test/internal/node/clientSecretCredential.spec.ts

```diff
@@ -60,17 +60,12 @@ describe("ClientSecretCredential (internal)", function() {
   // To test this, please install @azure/msal-node-extensions and un-skip these tests.
   describe("Persistent tests", function() {
     try {
-      /* eslint-disable-next-line @typescript-eslint/no-require-imports */
-      require("@azure/msal-node-extensions");
+      requireMsalNodeExtensions();
     } catch (e) {
       return;
```

Comment:
Won't this result in us ignoring errors and the tests that follow here not being run without us ever knowing it?

Reply:
requireMsalNodeExtensions() throws if we're on Node 8 🤔

Here's more info on this change: https://github.com/Azure/azure-sdk-for-js/pull/14702#discussion_r609910969

Reply:
Yes, but catching all errors and ignoring them seems dangerous.

The error is expected when running on node 8, but not otherwise. What this code tells me is that regardless of what reason `requireMsalNodeExtensions` throws, we are absolutely fine with it and will skip testing further

Reply:
Yes, that is correct for now. Since v2 beta release, these tests have been effectively skipped. The only way to run them was in Node 10, 12, 14 but only after having installed msal-node-extensions manually. These tests can't run if msal-node-extensions isn't installed and we removed it out of our dependencies. So, we'll only be able to enable these tests after Will's work.

Reply:
Understood.